source/configuration.txt

```diff
@@ -0,0 +1,16 @@
+TODO: write
```

Comment:
[Here are](https://github.com/BorisDog/mongo-csharp-analyzer/blob/vs20/samples/BasicSample/mongodb.analyzer.json)  all the configuration settings.

[This is how ](https://github.com/BorisDog/mongo-csharp-analyzer/blob/vs20/samples/BasicSample/BasicSample.csproj#L8-L10) the configuration file can be added.

Reply:
This is super helpful! Thank you for sharing. I've linked the preceding comment [in the ticket for writing](https://jira.mongodb.org/browse/DOCSP-19708) the Configuration page. We try to tie individual pages to their own Pull Requests to help keep tasks well defined and focused.

Are you alright adding this content in an upcoming PR?